backend/models/user.js

```diff
@@ -0,0 +1,27 @@
+const { connection } = require("../conf");
+const { ErrorHandler } = require("../utils/ErrorHandler");
+
+class UserModel {
```

Comment:
Like you don't use props or state you can build your function in function and not in class